infrastructure/terraform/functions/outputs.tf

```diff
@@ -1,15 +1,15 @@
 output "functionapp_endpoint_base" {
-  value = azurerm_function_app.fxn.default_hostname
+  value = azurerm_windows_function_app.fxn.default_hostname
```

Comment:
Why the name "azurerm_windows_function_app" instead of "azurerm_function_app" ? Is it running in Windows container?

Reply:
Because it is deprecated. See https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/function_app
Yes, I checked on a previous deployment (dubcard), it runs on Windows.